### PR TITLE
SOEOP-000: Upgrade Drupal Core 7.75

### PR DESCRIPTION
**Pull request by mdyoung3**
# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- Incorrectly reference Standford module in make file. Drush make failed to pull in the correct module. 

# Review By (Date)
12/3

# Criticality
10/10

# Review Tasks

## Setup tasks and/or behavior to test

1. Clone repo and run grunt
2. Check for modules files in codebase. 
3. Check logs for any mention of missing files.

### CODE REVIEW

File: soe/src/stanford.make [hunk 1]
```diff
@@ -152,7 +152,7 @@ projects[stanford_manage_content][download][url] = "git@github.com:SU-SWS/stanfo
 projects[stanford_manage_content][subdir] = "stanford"
 projects[stanford_manage_content][type] = "module"
 
-projects[stanford_news][download][tag] = "7.3.7-alpha5"
+projects[stanford_news][download][tag] = "7.x-3.7-alpha5"
```
Comment (sherakama) [inline soe/src/stanford.make:155]: Tag name for semver was `7.3.7-alpha.5` but `7.x-3.7-alpha5` is also available

**Review by sherakama — APPROVED** [review]
GTG

File: soe/src/stanford.make (hunk 1 above)
Comment (sherakama) [reply soe/src/stanford.make:155]: https://github.com/SU-SWS/stanford_news/releases/tag/7.3.7-alpha.5
https://github.com/SU-SWS/stanford_news/releases/tag/7.x-3.7-alpha5

**Review by boznik — APPROVED** [review]
Looks GTG

**Pull request merged**